CMakeLists.txt

```
@@ -15,13 +14,17 @@ ExternalProject_Add(cobra-3.6
   PREFIX cobra-3.6
   GIT_REPOSITORY https://github.com/nimble-code/cobra
   GIT_TAG ab06e676f9123ec07011a09c38daffaa4fbfdc60
-  INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/cobra_install"
+  INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/bin_linux"
```

What change is this intended to achieve? The install target from Cobra's makefiles is not configurable so the install dir here doesn't help us much. I set it by convention but haven't yet made it do anything.

If the intent is to target the bin_linux directory that cobra actually installs to it should probably be
```suggestion
  INSTALL_DIR <SOURCE_DIR>/bin_linux
```

---

I had a misunderstanding about INSTALL_DIR. Since it's not actually used in this case to install the binary, I'll revert to match convention.